### ISSUE TITLE
error to install ls-remote

### ISSUE THREAD

**Issue by enieber** [JulesGuesnon/LightRouter#1]
I recently started studying the Revery and this lib I wanna use, but when I tried to install using esy has the following error:

```shell
error: command failed: 'git' 'ls-remote' 'https://github.com/bryphe/lru%3Alru.opam.git' '2708c70'
stderr:
         fatal: unable to access 'https://github.com/bryphe/lru%3Alru.opam.git/': The requested URL returned error: 400
         
stdout:
         
  resolving @opam/lru@github:bryphe/lru%3Alru.opam#2708c70
  resolving revery@0.30.0
  resolving re-light-router@0.1.0
```

**Comment by JulesGuesnon** [comment]
Hello !
Right now, I'm in holidays, so I won't be able to fully help you.
I only noticed that your version is the 0.1, and I released the 1.0.

So you probably need to upgrade your dependencies

**Comment by JulesGuesnon** [comment]
Hey @enieber ! Did you achieve to fix your issue ?
